lib/dokumi/support/xml.rb

```diff
@@ -0,0 +1,17 @@
+module Dokumi
+  module Support
+    module XML
+      def self.read_xml_tag(text)
+        h = {}
+        s = StringScanner.new(text)
```

Comment:
As slightair was saying, you should use Nokogiri.
The reason `UnchangedStoryboardFinder` was parsing the XML without using Nokogiri was because it was working on diffs, not on whole files. `MisplacedConstraintFinder` should work on whole files.